client/src/main/java/com/battleships/gamewindow/services/RandomFleetPlacement.java

```diff
@@ -0,0 +1,124 @@
+package com.battleships.gamewindow.services;
+
+import com.battleships.models.board.Coordinate;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class RandomFleetPlacement {
+
+    private int lastCombinationProvided;
+    private Map<Integer, List<Coordinate>> boards;
+
+    public RandomFleetPlacement() {
+        this.boards=new HashMap<>();
+        this.boards = fillMap();
+        lastCombinationProvided = 0;
+    }
+
+    private List<Coordinate> getHardcodedCordsVersion1() {
+        return Arrays.asList(
+                Coordinate.fromIntCoords(4, 4),Coordinate.fromIntCoords(4, 5),Coordinate.fromIntCoords(4, 6),Coordinate.fromIntCoords(4, 7),
+                Coordinate.fromIntCoords(1, 6),Coordinate.fromIntCoords(1, 7),Coordinate.fromIntCoords(1, 8),
+                Coordinate.fromIntCoords(6, 3),Coordinate.fromIntCoords(7, 3),Coordinate.fromIntCoords(8, 3),
+                Coordinate.fromIntCoords(6, 5),Coordinate.fromIntCoords(7, 5),
+                Coordinate.fromIntCoords(9, 5),Coordinate.fromIntCoords(10, 5),
+                Coordinate.fromIntCoords(9, 8),Coordinate.fromIntCoords(9, 9),
+                Coordinate.fromIntCoords(5, 1),
+                Coordinate.fromIntCoords(2, 2),
+                Coordinate.fromIntCoords(7, 7),
+                Coordinate.fromIntCoords(6, 10)
+                );
+    }
+
+    private List<Coordinate> getHardcodedCordsVersion2() {
+       return Arrays.asList(
+                Coordinate.fromIntCoords(1, 1), Coordinate.fromIntCoords(1, 2), Coordinate.fromIntCoords(1, 3), Coordinate.fromIntCoords(1, 4),
+                Coordinate.fromIntCoords(4, 2), Coordinate.fromIntCoords(5, 2), Coordinate.fromIntCoords(6, 2),
+                Coordinate.fromIntCoords(6, 7), Coordinate.fromIntCoords(6, 8), Coordinate.fromIntCoords(6, 9),
+                Coordinate.fromIntCoords(9, 1), Coordinate.fromIntCoords(9, 2),
+                Coordinate.fromIntCoords(8, 9), Coordinate.fromIntCoords(8, 10),
+                Coordinate.fromIntCoords(3, 6), Coordinate.fromIntCoords(3, 7),
+                Coordinate.fromIntCoords(1, 8),
+                Coordinate.fromIntCoords(4, 9),
+                Coordinate.fromIntCoords(5, 5),
+                Coordinate.fromIntCoords(9, 6));
+
+    }
+
+    private List<Coordinate> getHardcodedCordsVersion3() {
+        return  Arrays.asList(
+                Coordinate.fromIntCoords(6, 8), Coordinate.fromIntCoords(7, 8), Coordinate.fromIntCoords(8, 8), Coordinate.fromIntCoords(9, 8),
```

Comment:
Please use iteration instead of repetition, pseudo code:
for (Coords coords : coordinates){
Coordinate.fromIntCoords(coords); }

Reply:
We will do it in next BattleShips, thanks. I mean - This is not worth to modify. If we would like to improve it, we will implement fully random ship placement generator and delete it all